EIPS/eip-7441.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7441
+title: Upgrade block proposer election to Whisk
+description: Allow elected block proposers to remain private until block publishing, to prevent DoS attacks
+author: George Kadianakis (@asn-d6), Justin Drake (@JustinDrake), dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7441-upgrade-block-proposer-election-to-whisk-ssle/15316
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+---
+
+## Abstract
+
+Upgrades the block proposer election mechanism to Whisk, a single secret leader election (SSLE) protocol. Currently, block proposers are publicly known in advance, sufficiently to allow sequential DoS attacks that could disable Ethereum. This upgrade makes the next block proposer unknown until its block is published.
+
+## Motivation
+
+The beacon chain currently elects the next 32 block proposers at the beginning of each epoch. The results of this election are public and everyone gets to learn the identity of those future block proposers.
+
+This information leak enables attackers to launch DoS attacks against each proposer sequentially in an attempt to disable Ethereum.
+
+## Specification
+
+### Execution layer
+
+This requires no changes to the Execution Layer.
+
+### Consensus layer
+
+The protocol can be summarized in the following concurrent steps:
+
```

Comment:
```suggestion
- Proposals are now divided into shuffling phases of `WHISK_EPOCHS_PER_SHUFFLING_PHASE` epochs
```

Reply:
I'm not sure if this sentence is necessary. TBH this mini summary of the protocol is not necessary either. It does not provide any specification and is only a general overview for readers. Should it be removed altogether?

Reply:
so the flow of someone looking into this feature would be (or atleast how we want it to be) : Read EIP => Read consensus specs => get into client implementations if someone wants to follow through on this. So mini summary helps making that jump

keeping that in mind, what kind of mini summary would you deem good to fit here. else if we have that somewhere in consensus specs (`fork.md`?) then we can refer it here i guess

Reply:
I don't think such summary would fit anywhere in the specs. Since it's derived information it mostly fits here or in a blog / etheresearch post introduction.